lib/screens/explore_screens/vote_screens/proposals/usecases/get_proposals_data_use_case.dart

```diff
@@ -13,14 +13,28 @@ class GetProposalsDataUseCase {
   final ProposalsRepository _proposalsRepository = ProposalsRepository();
 
   Future<List<Result>> run(ProposalType proposalType) {
-    final futures = [
-      _profileRepository.getProfile(settingsStorage.accountName),
-      _proposalsRepository.getProposals(proposalType),
-      _proposalsRepository.getReferendums(proposalType),
-      _proposalsRepository.getSupportLevel(_alliance),
-      _proposalsRepository.getSupportLevel(_campaign),
-      _proposalsRepository.getSupportLevel(_milestone),
-    ];
+    late final List<Future<Result>> futures;
+    if (proposalType.referendumStage.length > 1) {
```

Comment:
Please name the 1 into a constant with a meaningful name to help understand this logic.

Reply:
Nice suggestion